modules/renderer/src/text-measurer.ts

```diff
@@ -34,17 +34,19 @@ export default class TextMeasurer {
     this.ctx.font = `${fontStyle} ${fontWeight} ${fontSize}px ${fontFamily}`
   }
 
-  measure(text: string): Size {
-    const {fontSize, lineHeight} = this.opts
-    const rowHeight = fontSize * 1.2
-    const rowSpacing = fontSize * (lineHeight - 1)
+  measure(text: string, fontSize: number, fontFamily: string, fontStyle = 'normal'): Size {
```

Comment:
`measure(text:string, opts: Partial<TextMeasurerOptions>)`

Don't use unnamed parameters. They are impossible to read.

Reply:
fixed